components/nimbus/CHANGELOG.md

```diff
@@ -0,0 +1,161 @@
+# Unreleased changes
```

Comment:
Not for this PR, but I feel like it's going to be weird to have this nimbus-specific changelog here when future entries are going into the main changelog file. Some options:

* We could delete it.
* We could rename it to something like "OLD_CHANGELOG.md"
* We could painstakingly merge the changelog entries here into the main changelog, corresponding to the appservices version that they shipped in.

I think I'd probably lean towards renaming it. Thoughts?

(Just to reinforce, that doesn't have to block landing)

Reply:
Renamed and added an explanatory comment to the head of the file.